events-processor/cache/billable_metric_filters_consumer.go

```diff
@@ -0,0 +1,36 @@
+package cache
+
+import (
+	"context"
+
+	"github.com/getlago/lago/events-processor/models"
+	"github.com/getlago/lago/events-processor/utils"
+)
+
+func (c *Cache) StartBillableMetricFiltersConsumer(ctx context.Context) error {
+	return startGenericConsumer(ctx, c, ConsumerConfig[models.BillableMetricFilter]{
```

Comment:
Maybe we could use an interface instead of a config but I don't know if we can have generic interface.